src/pages/ni/config/RhConfig.vue

```diff
@@ -521,6 +747,107 @@ export default {
       if (!obj.name.required) return REQUIRED_LABEL;
       else if (!obj.name.sector) return 'Nom déjà existant';
     },
+    async getEstablishments () {
+      try {
+        this.establishments = await this.$establishments.list();
+      } catch (e) {
+        console.error(e);
+        this.establishments = [];
+        NotifyNegative('Erreur lors de la récupération des établissements.')
+      }
+    },
+    resetEstablishmentCreationModal () {
+      this.newEstablishment = {
+        name: '',
+        siret: '',
+        address: {},
+        phone: '',
+        workHealthService: '',
+        urssafCode: '',
+      };
+      this.establishmentCreationModal = false;
+      this.$v.newEstablishment.$reset();
+    },
+    async createNewEstablishment () {
+      try {
+        const isValid = await this.waitForFormValidation(this.$v.newEstablishment);
+        if (!isValid) return NotifyWarning('Champ(s) invalide(s)');
+
+        this.loading = true;
+        await this.$establishments.create(this.newEstablishment);
+        NotifyPositive('Établissement créé.');
+        this.resetEstablishmentCreationModal();
+        await this.getEstablishments();
+      } catch (e) {
+        console.error(e);
+        if (e.status === 409) {
+          return NotifyNegative(e.data.message);
+        }
+        return NotifyNegative("Erreur lors de la création de l'établissement");
+      } finally {
+        this.loading = false;
+      }
+    },
+    openEstablishmentEditionModal (establishmentId) {
+      this.editedEstablishment = this.$_.cloneDeep(
+        this.establishments.find(est => est._id === establishmentId) || this.editedEstablishment
```

Comment:
avec la position de ton `||`,  on peut cloneDeep `this.editedEstablishment` dans lui même ?

Reply:
en effet '-_-